client/src/app/domain/models/meetings/meeting.ts

```diff
@@ -462,17 +462,16 @@ export class Meeting extends BaseModel<Meeting> {
         `list_of_speakers_countdown_id`,
         `poll_countdown_id`,
         `projection_ids`,
-        `default_projector_agenda_all_items_ids`,
```

Comment:
this was just removed here and is nor refferenced anywhere else. 
Was this accidentially deleted?